MobileCenterCrashes/MobileCenterCrashesTests/MSCrashesTests.mm

```diff
@@ -14,6 +14,7 @@
 #import "MSServiceAbstractPrivate.h"
 #import "MSServiceAbstractProtected.h"
 #import "MSUtil.h"
+#import "MSCrashesUtil.h"
```

Comment:
You probably need one more unit test that verifies your implementation that `PriorityHigh` should go first.